Source/JavaScriptCore/runtime/JSSet.h

```diff
@@ -57,7 +57,7 @@ class JSSet final : public OrderedHashSet {
     }
 
     static bool isAddFastAndNonObservable(Structure*);
-    JS_EXPORT_PRIVATE bool isIteratorProtocolFastAndNonObservable();
```

Comment:
To address https://github.com/WebKit/WebKit/pull/50121#discussion_r2321126712 for Set